src/rgw/rgw_admin.cc

```diff
@@ -97,7 +97,8 @@ void usage()
   cout << "  subuser rm                 remove subuser\n";
   cout << "  key create                 create access key\n";
   cout << "  key rm                     remove access key\n";
-  cout << "  bucket list                list buckets\n";
+  cout << "  bucket list                list buckets (specify --allow-unordered for\n";
+  cout << "                             faster, unsorted listing)\n";
```

Comment:
help.t in src/test/cli/radosgw-admin needs to have the cli changes reflected.

Reply:
Thanks!